azure-pipelines.yml

```diff
@@ -0,0 +1,63 @@
+trigger:
+  branches:
+    include: [ '*' ]
+    exclude: [ 'refs/tags/*' ]
```

Comment:
This keeps it from triggering on tags, which I found surprising in another project where a release pipeline tagged the sources as the last stop of doing a release.